zmon-controller-ui/views/dashboard.html

```diff
@@ -65,7 +65,13 @@
     <!-- Alerts -->
     <div class="zmon-alert" ng-repeat="alertInstance in alerts | filter:alertSearch | orderBy:['alert_definition.priority', '-oldestStartTime'] track by $id(alertInstance.alert_definition.id)" ng-class="(compact ? 'col-md-6' : 'row').concat(' alert-prio-' + alertInstance.alert_definition.priority)" ng-hide="alertInstance.alert_definition.priority === 10 && !showAlertsWithAllEntitiesInDowntime">
         <div ng-class="compact ? 'col-md-10' : 'col-md-7'">
-            <h1><span class="fa fa-exclamation-triangle" title="Check execution for all entities that are in ALERT state now resulted in an exception!" ng-show="{{alertInstance.inException}}" /><a ng-href="#/alert-details/{{alertInstance.alert_definition.id}}">{{alertInstance.message}}</a> <span ng-if="alertInstance.numEntitiesNotInDowntimeNow > 1">({{alertInstance.numEntitiesNotInDowntimeNow}})</span></h1>
+            <h1>
+              <span class="fa fa-exclamation-triangle" title="Check execution for all entities that are in ALERT state now resulted in an exception!" ng-show="{{alertInstance.inException}}"></span>
+              <span class="fa fa-bell clickable" title="This alert has a notification" ng-show="!alertInstance.alert_definition.notifications_ack" ng-if="alertInstance.alert_definition.notifications.length" notification-modal alert="alertInstance"></span>
```

Comment:
Can we move the `notifications_ack` member to the `alertInstance` instead of the nested member `alertDefinition`?